.github/workflows/run_all_benchmarks.yml

```diff
@@ -77,19 +84,31 @@ jobs:
         run: |
           source ${HOME}/.cargo/env
           RUSTC_BOOTSTRAP=1 cargo build --profile production --features runtime-benchmarks --verbose
+      - name: create-chainspec
+        run: |
+          ${{ github.workspace }}/target/production/manta build-spec --chain $CHAIN_SPEC --disable-default-bootnode --raw > ${{ github.workspace }}/tests/data/fork.json
+      - name: append manta-pay storage
+        run: |
+          wget -P ${{ github.workspace }}/tests/data https://manta-ops.s3.amazonaws.com/integration-tests-data/storage.json
+          cd ${{ github.workspace }}/tests
+          yarn install
+          yarn
+          node append_storage.js
```

Comment:
are we sure this appends to `Manta/tests/data/fork.json` and not `Manta/data/fork.json`

Reply:
there's a `cd tests/` command